.github/workflows/dependencies.yml

```diff
@@ -0,0 +1,47 @@
+name: dependencies
+
+on:
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  upgrade:
+    name: Test dependencies
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python: [3.6, 3.8]
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Checkout default branch
+        uses: actions/checkout@v3
+        with:
+          ref: master
+          path: teuthology
+      - name: virtualenv
+        run: |
+          pip install --user virtualenv
+          virtualenv ./virtualenv
+          cd ./virtualenv/lib/python*
+          touch no-global-site-packages.txt
+        working-directory: ./teuthology
```

Comment:
The above step is temporarily needed until we remove or fix `bootstrap`'s behavior of removing virtualenvs that use system site-packages, as github actions seems to only want to create that type of virtualenv.